### PR TITLE
MINOR: Fix render order of extruded building depth pre pass

### PR DESCRIPTION
**Pull request by ninok**

### CODE REVIEW

**Comment by ninok** [comment]
@heremaps-bot retest this please

File: @here/harp-mapview/lib/DepthPrePass.ts [hunk 1]
```diff
@@ -120,7 +120,7 @@ export function createDepthPrePassMesh(mesh: THREE.Mesh): THREE.Mesh {
             : createDepthPrePassMaterial(mesh.material);
 
     const depthPassMesh = new THREE.Mesh(depthPassGeometry, depthPassMaterial);
-    depthPassMesh.renderOrder = mesh.renderOrder;
+    depthPassMesh.renderOrder = mesh.renderOrder - 1e-6;
```
Comment (kkostecki) [inline @here/harp-mapview/lib/DepthPrePass.ts]: Please enclose in some named const.

**Comment by ninok** [comment]
> It would be nice to mention in the commit message why this regression happened.
It is not a regression. Basically it was never working properly and it is really surprising that we never saw it before.

File: @here/harp-mapview/lib/DepthPrePass.ts [hunk 2]
```diff
@@ -66,14 +71,8 @@ export function createDepthPrePassMaterial(baseMaterial: THREE.Material): THREE.
...
  *
- * About render order: the DepthPrePass object that is created has the same `renderOrder` as
- * the original mesh. The proper sort orders of both the depth and the color pass objects are
- * guaranteed by ThreeJS's handling of transparent objects, which renders them after opaque
- * objects:
- *   - since the depth prepass object is never transparent, it is rendered first
- *   - since the color pass object is transparent, it is rendered second
- *
- * @see [Material.transparent in ThreeJS's doc](https://threejs.org/docs/#api/harp-materials/Material.transparent).
+ * The DepthPrePass object is created wis a slightly smaller `renderOrder` as the original mesh
```
Comment (kkostecki) [inline @here/harp-mapview/lib/DepthPrePass.ts]: Minor typo.

**Pull request merged**